templates/editConn.js

```diff
@@ -1,12 +1,12 @@
 const vscode = acquireVsCodeApi();
-let err = [];
+let hasErr = false;
```

Comment:
this global state is a little out of place, would the logic in this file be better encapsulated with a class?